ext/vscode/package.json

```diff
@@ -38,6 +38,11 @@
                 "command": "azure-dev.commands.cli.init",
                 "title": "%azure-dev.commands.cli.init.title%"
             },
+            {
```

Comment:
We should probably hide this command from the command palette, so that it doesn't cause confusion by seeing two entries for init. You can do that by adding an entry in `contributes.menus.commandPalette`. See `azure-dev.commands.getDotEnvFilePath` for an example.

Reply:
Thanks @bwateratmsft! Without hiding it I can see:

<img width="610" alt="image" src="https://github.com/user-attachments/assets/a851a896-84d2-4b06-abc2-f918c39c071e" />

But after adding the following to `contributes.menus.commandPalette`, there's only one entry now:

```json
 {
   "command": "azure-dev.commands.cli.initFromPom",
   "when": "false"
 },
```

<img width="667" alt="image" src="https://github.com/user-attachments/assets/9ff566de-454d-4ca6-bed4-150bddd3f6bd" />